monkey/infection_monkey/network/relay/utils.py

```diff
@@ -90,8 +89,7 @@ def send_remove_from_waitlist_control_message_to_relays(servers: Iterable[str]):
 
 
 def _send_remove_from_waitlist_control_message_to_relay(server: str):
-    ip, port = address_to_ip_port(server)
-    server_address = SocketAddress(IPv4Address(ip), int(port))
+    server_address = SocketAddress(*address_to_ip_port(server))
```

Comment:
Note: This is also modified in another PR, because the port from `address_to_ip_port` can be `None`. However, SocketAddress will validate the input because it is a pydantic model, which is also a valid solution.
This will change when the previous comment is addressed.

Reply:
Which PR? Anyhow, this function has been removed now.